README.md

```diff
@@ -0,0 +1,58 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Restaurant App
+
+It's a Restaurant app to show the restaurant, the menu and to contact the owner information dynamically by clicking in the button in the Nav Bar
+
+![screenshot](./assets/screenshot/screenshot.jpg)
```

Comment:
- [ ] [MANDATORY] This image link is invalid, kindly fix it. 
       ![Screen Shot 2021-06-21 at 02 06 00](https://user-images.githubusercontent.com/45217364/122692518-d7c45500-d235-11eb-8628-525ddddbabb7.png)

Reply:
Done! Thanks for the review.